src/main/webapp/city.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * @param {query} id
+ */
+async function searching(id) {
+  var cityId = null;
+
+  const citiesResponse = await fetch("/api/city.html?id=" + id);
+  const cities = await citiesResponse.json();
+
+  for (let city in cities) {
+    if ((cities[city]["id"] = id)) {
+      cityId = cities[city]["id"];
+    }
+  }
+
+  const eventsResponse = await fetch("/api/events");
+  const events = await eventsResponse.json();
+  var eventName = [];
+  var eventDescription = [];
+  var eventDate = [];
+  var eventLocation = [];
+  var eventPricing = [];
+
+  for (let event in events) {
+    if ((events[event]["cityId"] = cityId)) {
+      eventName.push(events[event]["name"]);
+      eventDescription.push(events[event]["description"]);
+      eventDate.push(events[event]["date"]);
+      eventLocation.push(events[event]["location"]);
+      eventPricing.push(events[event]["pricing"]);
+    }
+  }
+  displayCityResults(
+    cities[city],
+    eventName,
+    eventDescription,
+    eventDate,
+    eventLocation,
+    eventPricing
+  );
```

Comment:
Per the [API](https://stoplight.io/p/docs/gh/gferioli0418/travelbud/reference/Travelbud.v1.yaml/paths/~1api~1cities~1%7BcityID%7D/get?srn=gh/gferioli0418/travelbud/reference/Travelbud.v1.yaml/paths/~1api~1cities~1{cityID}/get&group=api-schema) `city.events` should already contain the events for that city. So I would just do:
```suggestion
  displayCityResults(city);
```